website/package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "website",
+  "packageManager": "yarn@3.0.0",
+  "peerDependencies": {
+    "@babel/cli": "^7.8.4",
```

Comment:
Probably should just be `*` range since it'll use the root workspaces versions of the same package.